packages/kit/src/core/dev/index.js

```diff
@@ -215,7 +215,7 @@ class Watcher extends EventEmitter {
 							hooks: {
 								getSession: hooks.getSession || (() => ({})),
 								handle: hooks.handle || (({ request, resolve }) => resolve(request)),
-								serverFetch: hooks.serverFetch || fetch
+								serverFetch: hooks.serverFetch || globalThis.fetch
```

Comment:
What about the fetch call inside `serverFetch`? Does that also need to be scoped with `globalThis.`, or does this only affect the default case of not having `serverFetch` defined?

Reply:
no, it's fallback, `serverFetch` is optional.

Reply:
> or does this only affect the default case of not having serverFetch defined?

yes.